recipes/meshplex/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "meshplex" %}
+{% set version = "0.13.4" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
```

Comment:
You can remove the quites in the jinja here and above. Also in the script line below.

Reply:
Done